### PR TITLE
feat: #170 update models

### PR DESCRIPTION
**Pull request by Howl404**

### CODE REVIEW

**Review by SergioAJS — APPROVED** [review]

File: mw-webapp/src/logic/waysTable/columns.tsx [hunk 1]
```diff
@@ -21,21 +21,20 @@ export const OWNER_EMAIL = "Owner's email";
...
 export const columns = [
   columnHelper.accessor<"uuid", string>("uuid", {
-    // Maybe we should add way's name? Need to think)
     header: "Way's uuid",
 
     /**
      * Cell with owner's name
      */
     cell: ({row}) => renderCellValue(row.original.uuid),
   }),
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/waysTable/columns.tsx:30]: Maybe we no need render uuid of way in the table

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/logic/waysTable/columns.tsx (hunk 1 above)
Comment (Howl404) [reply mw-webapp/src/logic/waysTable/columns.tsx:30]: I think Victor said that it might be there

Comment (Ekaterina1994) [reply mw-webapp/src/logic/waysTable/columns.tsx:30]: If you mean this video QA session https://www.youtube.com/watch?v=fLDg2SLUcWg

Then Viktor speak about it on 2.26-2.35 .

Comment (Ekaterina1994) [reply mw-webapp/src/logic/waysTable/columns.tsx:30]: If you think then we will need ID then you can don't delete, but then add comment for delete it later

Comment (Howl404) [reply mw-webapp/src/logic/waysTable/columns.tsx]: Thanks, deleted

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]
Yeah) that's what we need

**Pull request merged**